diff --git a/dist/lib/ScheduledTaskHandler.js b/dist/lib/ScheduledTaskHandler.js
index 637b084c105d2b2b4fc827149bf6032e2f8deabb..7e3d79ba605eb2decd844445060744ba142c2ebc 100644
--- a/dist/lib/ScheduledTaskHandler.js
+++ b/dist/lib/ScheduledTaskHandler.js
@@ -121,7 +121,7 @@ class ScheduledTaskHandler {
             const taskRunResult = await piece.run(payload);
             const { duration } = stopwatch.stop();
             framework_1.container.client.emit(ScheduledTaskEvents_1.ScheduledTaskEvents.ScheduledTaskSuccess, task, payload, taskRunResult, duration);
-            return duration;
+            return taskRunResult;
         });
         result.inspectErr((error) => framework_1.container.client.emit(ScheduledTaskEvents_1.ScheduledTaskEvents.ScheduledTaskError, error, task, payload));
         const value = result.unwrapOr(null);
